src/renderer/pages/App.js

```diff
@@ -82,8 +82,9 @@ class App extends React.Component {
     return (
       <div key='errors'
         className={'error-popover ' + (hasErrors ? 'visible' : 'hidden')}>
-        <div key='title' className='title'><FormattedMessage id='error'
-          defaultMessage='Error'/></div>
```

Comment:
`defaultMessage` is part of the [React-Intl Message Formatting Fallbacks](https://github.com/yahoo/react-intl/wiki/API#message-formatting-fallbacks), this ensures a message is always retrieved (think of incomplete translation). We could load an english lang file and overwrite it with the local labels but that is not the way the library is designed.

I find it convenient in code navigation and readability to have the labels in the code but if you prefer we can setup this in another way.

What do you think? Can you have a look at my issue refreshing the view? I am a bit stuck ATM...